### ISSUE TITLE
Getting Started - module joy/db error (Joy 0.7.2)

### ISSUE THREAD

**Issue by sperelson** [joy-framework/joy#45]
Followed the Getting Started in the readme and got an error after creating the `account` route:
```
error: could not find module joy/db
``` 
Resolved by changing the import in `src/routes/account.janet`:
```
- (import joy/db)
+ (import db)
```

Running on a Mac (Mojave 10.14.6). Joy Version: 0.7.2. Janet Version:1.8.1

Thanks 🙌🏻

**Comment by swlkr** [comment]
So yeah I just made this change, didn't realize I was going to get a slew of new people using it 😅

Two other things that would work for the generated routes are just deleting `(import joy/db)` and making sure `(import joy :prefix "")` or `(use joy)` is there, that will give you the `db/` functions as well

**Comment by swlkr** [comment]
Closed by 06b1316439e2840074d753f1efa42eac6d3cd1be
